src/main/java/com/josdem/vetlog/model/Pet.java

```diff
@@ -14,84 +14,96 @@
-    private Breed breed;
-
-    @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
-    @JoinColumn(name = "pet_id")
-    private PetAdoption adoption;
-
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "user_id")
-    private User user;
-
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "adopter_id")
-    private User adopter;
-
-    @OneToMany(fetch = FetchType.LAZY)
-    @JoinColumn(name = "pet_image_id")
-    private List<PetImage> images;
-
-    @Transient
-    private List<Vaccination> vaccines;
-}
+ package com.josdem.vetlog.model;
+
+ import static jakarta.persistence.EnumType.STRING;
+ import static jakarta.persistence.GenerationType.IDENTITY;
+ 
+ import com.josdem.vetlog.enums.PetStatus;
+ import jakarta.persistence.CascadeType;
+ import jakarta.persistence.Column;
+ import jakarta.persistence.Entity;
+ import jakarta.persistence.Enumerated;
+ import jakarta.persistence.FetchType;
+ import jakarta.persistence.GeneratedValue;
+ import jakarta.persistence.Id;
+ import jakarta.persistence.JoinColumn;
+ import jakarta.persistence.OneToMany;
+ import jakarta.persistence.OneToOne;
+ import jakarta.persistence.Transient;
+ import java.time.LocalDateTime;
+ import java.util.List;
+ import lombok.Getter;
+ import lombok.Setter;
+ import lombok.ToString;
+ 
+ @Getter
+ @Setter
+ @Entity
+ @ToString
+ public class Pet {
+ 
+     @Id
+     @GeneratedValue(strategy = IDENTITY)
+     private Long id;
+ 
+     @Column(nullable = false)
+     private String uuid;
+ 
+     @Column(nullable = false)
+     private String name;
+ 
+     @Column(nullable = false)
+     private LocalDateTime birthDate;
+ 
+     @Column(nullable = false)
+     private Boolean dewormed = false;
+ 
+     @Column(nullable = false)
+     private Boolean sterilized = false;
+ 
+     @Column(nullable = false)
+     private Boolean vaccinated = false;
+ 
+     @Column(nullable = false)
+     private LocalDateTime dateCreated = LocalDateTime.now();
+ 
+     @Column(nullable = false)
+     @Enumerated(STRING)
+     private PetStatus status;
+ 
+     @OneToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "breed_id")
+     private Breed breed;
+ 
+     @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+     @JoinColumn(name = "pet_id")
+     private PetAdoption adoption;
+ 
+     @OneToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "user_id")
+     private User user;
+ 
+     @OneToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "adopter_id")
+     private User adopter;
+ 
+     @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
+     @JoinColumn(name = "pet_id")
+     private List<VaccinationRecord> vaccinationRecords;
+ 
+     @OneToMany(fetch = FetchType.LAZY)
+     @JoinColumn(name = "pet_image_id")
+     private List<PetImage> images;
+ 
+     // New Methods for Vaccination Records
+ 
+     public void setVaccinationRecords(List<VaccinationRecord> vaccinationRecords) {
```

Comment:
@AviraLS22 , I don't know why the whole file change is getting shown here. 

But when you use getter and setter anotation, you don't need to explicitly add these methods.

Reply:
@yashudhlani , I’ve removed the manually added getter and setter methods for vaccinationRecords as per the feedback. Lombok's @Getter and @Setter annotations are now used instead.